docs/user_manual.md

```diff
@@ -272,7 +273,11 @@ For example, `{"foo": {"internalPort": 4711}}`  will  map the internal port
 4711 of the container to an arbitrary external port on the host. 
   
 Specifying `{"foo": {"internalPort": 4711, "externalPort": 80}}` will map
-internal port 4711  of the container to port 80 on the host. 
+internal port 4711  of the container to port 80 on the host.
+
+The `ip` attribute is optional and specifies the IP address on which the port
+will be exposed. Although Helios will check the IP string is a valid IP address,
+it won't verify the address is routable or whether it'll work at deploy-time.
```

Comment:
I've wondered what's the correct terminology here. Does one say a port is exposed on an IP and that an IP is bound to an interface?